### PR TITLE
Update eCommerce guide to match the existing implementation

### PR DESCRIPTION
**Pull request by daallen303**
Story: https://www.notion.so/centrapay/E-commerce-High-level-Documentation-b54f81e73ed74a228b882f4127ae593d?pvs=4

We need a guide that integrator's can use when exploring eCommerce. The current eCommerce guide wasn't useful so it has been completely redone. This is a starting point and this guide will be updated with more details in the following weeks.

Preview: http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/guides/ecommerce/

### CODE REVIEW

File: src/content/guides/ecommerce-website.mdx
Comment (daallen303) [inline src/content/guides/ecommerce-website.mdx:1]: This guide talks about redirecting users using a redirectUrl after completed payments. It looks like @Neeraj-Ghodla was the one who implemented redirect urls. Could you confirm if this is being used by anyone? To my knowledge it is not but it if is it should probably be moved somewhere else.

File: src/content/guides/ecommerce.mdx
Comment (laurencemcknight) [inline src/content/guides/ecommerce.mdx:13]: You could mention that the merchant needs a `e-commerce` type.

File: src/content/guides/ecommerce.mdx
Comment (laurencemcknight) [inline src/content/guides/ecommerce.mdx:14]: "Centrapay will provision these on behalf of the integrators" - We may want to encourage integrators to create merchants and merchant configs through the business portal.

File: src/content/guides/ecommerce.mdx
Comment (laurencemcknight) [inline src/content/guides/ecommerce.mdx:15]: Are you able to link the certification checklist on "getting certified"?

File: src/content/guides/ecommerce.mdx
Comment (daallen303) [reply src/content/guides/ecommerce.mdx:14]: Yeah maybe eventually I just didn't want to add anything that they can't currently do. I'm not sure what the evolution of e-commerce merchants might look like, I could see it being more locked down then creating physical merchants.

File: src/content/guides/ecommerce.mdx
Comment (daallen303) [reply src/content/guides/ecommerce.mdx:15]: I don't think we have a public list they can look at, we provide that for them when they contact us. We do have a core requirements section that we should add but I'm not 100% sure on the core requirements for eCommerce yet.

File: src/__tests__/__snapshots__/links.integration.test.js.snap
Comment (tom-blake) [inline src/__tests__/__snapshots__/links.integration.test.js.snap:318]: @daallen303 Change: Chuck a redirect in for this old url to avoid breaking this link

File: src/content/guides/ecommerce.mdx
Comment (tom-blake) [inline src/content/guides/ecommerce.mdx:11]: Suggestion: You could make this section more concise so that it's easier to read

Something like "Please contact Centrapay to get your business configured for ecommerce. Centrapay will provide you with API key to request payment(link)

Centrapay will configure your business to accept test payments before configuring you to accept live payments. Accepting live payments requires you to meet our certification requirements.

File: src/content/guides/ecommerce.mdx
Comment (tom-blake) [reply src/content/guides/ecommerce.mdx:15]: Yeah I think the Core Requirements would be pretty similar to the [POS requirements](https://docs.centrapay.com/guides/point-of-sale/)

Difference being an ecommerce payment flow instead of QR/barcode

Comment (laurencemcknight) [reply src/content/guides/ecommerce.mdx:15]: Ok, we can add those at a later date when the requirements are fully defined.

**Review by laurencemcknight — APPROVED** [review]
Approving to unblock. Please address Tom's comments before merging.

File: src/__tests__/__snapshots__/links.integration.test.js.snap
Comment (daallen303) [reply src/__tests__/__snapshots__/links.integration.test.js.snap:318]: I wasn't sure how to do a redirect so I just kept the filename the same. It's still a relevant title I was just trimming it to be a little neater.

File: src/content/guides/ecommerce.mdx
Comment (daallen303) [reply src/content/guides/ecommerce.mdx]: Updated it to match your suggestion. :)

**Pull request merged**
